Fixes error

dl-runtime.c:222: error: parse error before "CFI_STARTPROC"
make[2]: *** [crosstool-0.28-rc35/build/s390-unknown-linux-gnu/gcc-3.4.1-glibc-20040822/build-glibc/elf/dl-runtim
e.o] Error 1
make[2]: Leaving directory `crosstool-0.28-rc35/build/s390-unknown-linux-gnu/gcc-3.4.1-glibc-20040822/glibc-20040
822/elf'
make[1]: *** [elf/subdir_lib] Error 2
make[1]: Leaving directory `crosstool-0.28-rc35/build/s390-unknown-linux-gnu/gcc-3.4.1-glibc-20040822/glibc-20040
822'
make: *** [all] Error 2

-----------------

See http://sources.redhat.com/bugzilla/show_bug.cgi?id=398
http://sources.redhat.com/ml/libc-alpha/2004-07/msg00023.html

-----------------

CVSROOT:	/cvs/glibc
Module name:	libc
Branch: 	glibc-2_3-branch
Changes by:	roland@sources.redhat.com	2006-02-23 22:36:51

Modified files:
	sysdeps/s390/s390-64: dl-machine.h 
	sysdeps/s390/s390-32: dl-machine.h 

Log message:
	2004-07-10  GOTO Masanori  <gotom@debian.or.jp>
	
	[BZ #398]
	* sysdeps/s390/s390-32/dl-machine.h: Include <sysdep.h> for CFI
	directive.
	* sysdeps/s390/s390-64/dl-machine.h: Likewise.

Patches:
http://sources.redhat.com/cgi-bin/cvsweb.cgi/libc/sysdeps/s390/s390-64/dl-machine.h.diff?cvsroot=glibc&only_with_tag=glibc-2_3-branch&r1=1.19.4.1&r2=1.19.4.2
http://sources.redhat.com/cgi-bin/cvsweb.cgi/libc/sysdeps/s390/s390-32/dl-machine.h.diff?cvsroot=glibc&only_with_tag=glibc-2_3-branch&r1=1.20.4.1&r2=1.20.4.2

===================================================================
RCS file: /cvs/glibc/libc/sysdeps/s390/s390-64/dl-machine.h,v
retrieving revision 1.19.4.1
retrieving revision 1.19.4.2
diff -u -r1.19.4.1 -r1.19.4.2
--- libc/sysdeps/s390/s390-64/dl-machine.h	2005/10/17 04:57:27	1.19.4.1
+++ libc/sysdeps/s390/s390-64/dl-machine.h	2006/02/23 22:36:51	1.19.4.2
@@ -27,6 +27,7 @@
 #include <sys/param.h>
 #include <string.h>
 #include <link.h>
+#include <sysdep.h>
 
 /* This is an older, now obsolete value.  */
 #define EM_S390_OLD	0xA390
===================================================================
RCS file: /cvs/glibc/libc/sysdeps/s390/s390-32/dl-machine.h,v
retrieving revision 1.20.4.1
retrieving revision 1.20.4.2
diff -u -r1.20.4.1 -r1.20.4.2
--- libc/sysdeps/s390/s390-32/dl-machine.h	2005/10/17 04:57:27	1.20.4.1
+++ libc/sysdeps/s390/s390-32/dl-machine.h	2006/02/23 22:36:51	1.20.4.2
@@ -27,6 +27,7 @@
 #include <sys/param.h>
 #include <string.h>
 #include <link.h>
+#include <sysdep.h>
 
 /* This is an older, now obsolete value.  */
 #define EM_S390_OLD	0xA390
